### ISSUE TITLE
[RFE] Implement `check-dns` command

### ISSUE THREAD

**Issue by blanquicet** [Azure/kubectl-az#8]
## Current situation

If users want to check if DNS is working on a given node, they have to use the `run-command`:

```
$ kubectl az run-command "nslookup www.microsoft.com" --node aks-agentpool-21361837-vmss000000
Running...

[stdout]
Server:         168.63.129.16
Address:        168.63.129.16#53

Non-authoritative answer:
www.microsoft.com       canonical name = www.microsoft.com-c-3.edgekey.net.
www.microsoft.com-c-3.edgekey.net       canonical name = www.microsoft.com-c-3.edgekey.net.globalredir.akadns.net.
www.microsoft.com-c-3.edgekey.net.globalredir.akadns.net        canonical name = e13678.dscb.akamaiedge.net.
Name:   e13678.dscb.akamaiedge.net
Address: 104.123.41.162
Name:   e13678.dscb.akamaiedge.net
Address: 2a02:26f0:6d00:595::356e
Name:   e13678.dscb.akamaiedge.net
Address: 2a02:26f0:6d00:5a9::356e


[stderr]
```

## Ideal future situation

Investigate if it could be useful for users to have a dedicated command like `check-dns` which not just runs `nslookup` but additional commands that provide a more detailed output to better understand why a given URL wasn't resolved. 

Not being able to resolve the URL could be caused by many reasons. For instance, did we actually received a response for our request or it went in time-out? If it went in time-out, was it because the resolver never responded or because our firewall is blocking the DNS request or response? ... etc. 

In other words, how can we automate part of this investigation within a command?

**Comment by mqasimsarfraz** [comment]
Done as part of https://github.com/Azure/kubectl-aks/releases/tag/v0.3.0
